### ISSUE TITLE
allow to set title to main window or take title from document.title

### ISSUE THREAD

**Issue by marianoguerra** [mozilla/qbrt#120]
on ubuntu the main window doesn't display a title when not maximized, if I maximize it it displays 'Untitled window'

qbrt run --title "My Title" <url>

or take it from document.title

PS: This looks great for demos and screencasts where you don't want the chrome, your extensions, or other tabs from cluttering the demo.

**Comment by mykmelez** [comment]
Good point! Setting the window title to `document.title` seems like a reasonable solution—and a straightforward one to implement.
